ui/src/App.scss

```diff
@@ -181,3 +181,13 @@ body {
 .ant-modal-wrap {
   padding-top: $header-min-height;
 }
+
+.ant-form-item-label {
+  text-overflow: ellipsis;
+  z-index: 1;
+  padding-right: 8px !important;
```

Comment:
the screenshots are with these values?

Reply:
Yes, without this the padding is different when the ellipsis is used vs when the label fits (since Ant adds some padding after the colon).